ivc/src/ivc/interpreter.rs

```diff
@@ -913,15 +916,101 @@ where
 {
 }
 
+/// Builds selectors for the IVC circuit.
+pub fn build_selectors<F, const N_COL_TOTAL: usize, const N_CHALS: usize>(
+    domain_size: usize,
+) -> Vec<Vec<F>>
+where
+    F: PrimeField,
+{
+    // 3*N + 6*N+2 + N+1 + 35*N + 5 + 1 + N_CHALS =
+    // 45N + 9 + N_CHALS
+    let mut selectors: Vec<Vec<F>> = vec![vec![F::zero(); domain_size]; N_BLOCKS];
+    let mut cur_row = 0;
```

Comment:
Just a tiny nit, in this repo we normally use 4 letters for current, meaning we would normally name this `curr_row`, in case you want to make it all look uniform with the rest.

Reply:
Fixed!